src/fromager/commands/build.py

```diff
@@ -69,7 +69,7 @@ def build_sequence(
     with open(build_order_file, "r") as f:
         for entry in json.load(f):
             wheel_filename = _build(
-                wkctx, entry["dist"], entry["version"], sdist_server_url
+                wkctx, entry["dist"], Version(entry["version"]), sdist_server_url
```

Comment:
Maybe we can merge https://github.com/python-wheel-build/fromager/pull/212 instead, and you can keep this commit focused on the warning changes.

Reply:
okay, sounds good

Reply:
Let me know when you merge https://github.com/python-wheel-build/fromager/pull/212 or should I merge it?

Reply:
I think if you review and leave an approval it will merge.